pset-iterators.js

```diff
@@ -210,8 +256,18 @@ Example (not the actual answer):
 console.log('Problem 8:')
 
 // Add your code below this line
-
-
+const firstNameCount = presidents.reduce((firstNames,president) => {
+  let firstName = president.president.split(' ')[0]
+  if(firstNames[firstName] === undefined){
+    firstNames[firstName] = 1
+  }
+  else {
+    firstNames[firstName] += 1
+  }
+  return firstNames
+})
```

Comment:
Brilliant use of the reduce method to create an empty object here! One tip, you could set the initial incriminator of the reduce method after the function call (this would prevent a situation where you have an entire object turn into key/value pairs like in this example). [Here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L259) is an example of what I am referring to.